exercises/bowling/bowling.mli

```diff
@@ -9,4 +9,4 @@ of pins knocked down. The return value is the updated state of the game. *)
 val roll : int -> t -> t
 
 (** This is called at the end of a game to retrieve the final score. *)
-val score : t -> int
+val score : t -> (int, string) result
```

Comment:
There is a discussion on the interpretation of errors and how to handle them. I think it is up to the maintainers to find a suitable means of expressing the error situations.

Having said that, the choice of encoding errors as `-1`, is unfortunate in my eyes.

None the less, I am still in favour `result` instead of `option` because I feel that the score, if there is one, should be an `int`.